lncfg/sweeper.go

```diff
@@ -3,11 +3,24 @@ package lncfg
 import (
 	"fmt"
 	"time"
+
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
+)
+
+const (
+	// MaxFeeRateFloor is the smallest config value allowed for the max fee
+	// rate in sat/vb.
+	MaxFeeRateFloor = 100
+
+	// MaxAllowedFeeRate is the largest fee rate in sat/vb that we allow
+	// when configuring the MaxFeeRate.
+	MaxAllowedFeeRate = 10_000
 )
 
 //nolint:lll
 type Sweeper struct {
-	BatchWindowDuration time.Duration `long:"batchwindowduration" description:"Duration of the sweep batch window. The sweep is held back during the batch window to allow more inputs to be added and thereby lower the fee per input."`
+	BatchWindowDuration time.Duration        `long:"batchwindowduration" description:"Duration of the sweep batch window. The sweep is held back during the batch window to allow more inputs to be added and thereby lower the fee per input."`
+	MaxFeeRate          chainfee.SatPerVByte `long:"maxfeerate" description:"Maximum fee rate in sat/vb that the sweeper is allowed to use when sweeping funds."`
```

Comment:
I think we should also add the text `Setting this value too low can result in transactions not being confirmed in time, causing HTLCs to expire hence potentially losing funds.` here as not all users read the config file.
Will make it quite long, but better be safe than sorry (because people **WILL** set this to 1 sat/vByte and then open issues).

Reply:
Ah, I see, I missed that. Sounds good to me!